src/param.js

```diff
@@ -258,6 +258,40 @@ class GeoPoint extends NumberList {
 module.exports.GeoPoint = (...args) => new GeoPoint(...args);
 
 
+class GeoRect extends NumberList {
+  constructor(...args) {
+    super(...args);
+    this.minItems = 4;
+    this.maxItems = 4;
+  }
+
+  validate(value) {
+    let valid = super.validate(value);
+    if (valid) {
+      const valueParsed = (this.stringInput ? JSON.parse(value) : value);
+      if (
+        valueParsed.length !== 4
```

Comment:
Looks good, but shouldn't we be checking that these are all numbers? Same above for geopoint... I was thinking `valueParsed.every(e => typeof e === "number")`?

Reply:
That check is being done above via the parent class though, right? If there was an element in the list that wasn't a number, the result of `let valid = super.validate(value)` would be `false`